samples/contrib/pytorch-samples/bert/config.properties

```diff
@@ -4,7 +4,7 @@ metrics_address=http://0.0.0.0:8082
 enable_metrics_api=true
 metrics_format=prometheus
 number_of_netty_threads=4
-install_py_dep_per_model=true
+install_py_dep_per_model=false
```

Comment:
Setting it to `false` until Custom package installation issue is fixed in torchserve